lib/libutee/tee_api_objects.c

```diff
@@ -331,12 +433,28 @@ void TEE_CloseAndDeletePersistentObject(TEE_ObjectHandle object)
 	if (object == TEE_HANDLE_NULL)
 		return;
 
-	res = utee_storage_obj_del(object);
+	res = TEE_CloseAndDeletePersistentObject1(object);
```

Comment:
This change is not 1.1 compliant:
*If object is TEE_HANDLE_NULL, the function does nothing.*